src/index.js

```diff
@@ -51,9 +53,18 @@ export default class HillChart extends EventEmitter {
     this.chartHeight = height - margin.top - margin.bottom;
 
     // Render the svg and center chart according to margins
+    const colorScheme = this.darkMode ? 'hill-chart-dark' : 'hill-chart-light';
```

Comment:
```suggestion
    this.colorScheme = this.darkMode ? 'hill-chart-dark' : 'hill-chart-light';
```
We may need to use it on other places in the future